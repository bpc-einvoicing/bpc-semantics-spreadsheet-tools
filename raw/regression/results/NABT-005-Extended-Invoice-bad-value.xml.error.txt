1. Document Currency Code not specified or value specified is not allowed. /Invoice / bpc:codelist(cbc:DocumentCurrencyCode,'ISO-4217') (:NABT-005 Row 9 Tab 'Invoice, Credit Note':)
2. The first tax total is not in the document currency. /Invoice/cac:TaxTotal[1]/cbc:TaxAmount[1] / @currencyID = /*/cbc:DocumentCurrencyCode (:NABT-111 Row 186 Tab 'Invoice, Credit Note':)
Count of data errors: 2
