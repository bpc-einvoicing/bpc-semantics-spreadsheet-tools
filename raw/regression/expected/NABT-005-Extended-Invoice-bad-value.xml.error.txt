1. Document Currency Code not specified or value specified is not allowed. /Invoice / bpc:codelist(cbc:DocumentCurrencyCode,'ISO-4217') (:NABT-005 Row 9 Tab 'Invoice, Credit Note':)
Count of data errors: 1
